### DISCUSSION TITLE
guide/

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#15] [Announcements]
# guide/

Documentation for alist v3

https://alist.nn.ci/guide/

**Comment by Kimi-Arthur** [comment]
The guide doesn't have information to start a basic alist server...
This page is breif introduction, nit guide...

**Reply by Qq3135266742** [reply]
https://docs.qq.com/form/page/DUW1YVVJNbHpzV2No#/fill-detail

**Comment by DKA001** [comment]
alist获取的文件，为什么在域名后带有一个d文件？

示例：***.***.com/d/abc

**Reply by DKA001** [reply]
有朋友知道怎么回事吗？

**Reply by AIBUE** [reply]
是这样的

**Comment by vxth93** [comment]
Why are files not loading been trying to fix reloading page still not working

**Comment by Qq3135266742** [comment]
https://docs.qq.com/form/page/DUW1YVVJNbHpzV2No#/fill-detail

**Comment by Qq3135266742** [comment]
https://docs.qq.com/form/page/DUW1YVVJNbHpzV2No#/fill-detail

**Comment by Lonewolf1417** [comment]
Hi, im fairly new to tech and I appreciate all the help if have gotten from you guys.

**Comment by greatgithubfish** [comment]
pikpak网盘的令牌如何获取
